titus_isolate/utils.py

```diff
@@ -181,6 +185,26 @@ def get_cell_name():
         log.error("Failed to determine isolation cell.")
         return UNKNOWN_CELL
 
+def get_grpc_cell_name():
+    config_manager = get_config_manager()
+    if config_manager is None:
+        log.warning("Config manager is not yet set.")
+        return UNKNOWN_CELL
+
+    endpoint = config_manager.get_str(GRPC_REMOTE_ALLOC_ENDPOINT, None)
```

Comment:
If this function continues to exist, you probably want `get_cached_str` along with the inclusion in the "restart" list.